app/models/early_career_payments.rb

```diff
@@ -49,4 +49,8 @@ def first_eligible_qts_award_year(claim_year = nil)
   def last_ineligible_qts_award_year
     first_eligible_qts_award_year - 1
   end
+
+  def student_loan_balance_url
```

Comment:
@martikat There should be a spec to cover this method in the corresponding model spec. Could you add please?

Reply:
Added spec in spec/models/early_career_payments_spec.rb